notebooks/server_examples.ipynb

```diff
@@ -13,25 +13,41 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Error from server (AlreadyExists): namespaces \"seldon\" already exists\n"
+     ]
+    }
+   ],
    "source": [
```

Comment:
@adriangonz you sure want to commit notebook outputs ?

Reply:
Good point, let me remove the outputs.